tests.py

```diff
@@ -234,7 +238,7 @@ def test_read_message_too_long(self):
             with s.accept()[0] as k:
                 c.communicate(b"a" * (MAX_MESSAGE_LEN + 1))
                 sent = k.recv(2000)
-                self.assertEqual(sent, prepare_message(b"a" * 1000) + prepare_message(b"a"))
+                self.assertIn(sent, {prepare_message(b"a" * 1000) + prepare_message(b"a"), prepare_message(b"a"*1000)})
```

Comment:
Czy jak klient dostanie na wejściu linię dłuższą niż 1000 to resztę ma zignorować czy wysłać jako oddzielny komunikat?

Odp: Obojętnie (byleby nie wysłał komunikatu dłuższego niż 1000).